service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/RespondentSolicitorContentProvider.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.model.notify.submittedcase.RespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.SharedNotifyContentProvider;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+
+public class RespondentSolicitorContentProvider extends SharedNotifyContentProvider {
+
+    public RespondentSolicitorTemplate buildNotifyRespondentSolicitorTemplate(CaseData caseData,
+                                                                              RespondentSolicitor representative) {
+
+        return RespondentSolicitorTemplate.builder()
+            .representativeName(representative.getLastName())
+            .localAuthority(caseData.getCaseLocalAuthorityName())
```

Comment:
this is the LA code, it must be converted to name using LocalAuthorityNameLookupConfiguration which will check the code->name mapping

Reply:
nevermind, that is the name you pulled, ignore

Reply:
only concern is that for existing cases this may be null and so if in future we want to trigger this notification on an old case (post submission), I guess it may be empty